src/svg/graphic.ts

```diff
@@ -107,7 +108,7 @@ function bindStyle(svgEl: SVGElement, style: AllStyleOption, el?: Path | TSpan |
         // stroke then fill for text; fill then stroke for others
         attr(svgEl, 'paint-order', style.strokeFirst ? 'stroke' : 'fill');
         attr(svgEl, 'stroke-opacity', (style.strokeOpacity != null ? style.strokeOpacity * opacity : opacity) + '');
-        let lineDash = style.lineDash;
+        let lineDash = style.lineDash && normalizeLineDash(style.lineDash, el instanceof TSpan ? void 0 : style.lineWidth);
         if (lineDash) {
```

Comment:
Sincde normalizeLineDash will reject `0` lineWidth.

```js
el instanceof TSpan ? void 0 : style.lineWidth
```

Can be 

```js
el instanceof TSpan ? 0 : style.lineWidth
```

Reply:
Here I want the text element to ignore the judgment `lineWidth <= 0` and continue to invoke the bottom logic in `normalizeLineDash` function. If we pass `0` to it, the returned `lineDash` will be `null`, which is unexpected.

Reply:
I will remove this condition.